docs/advanced_topics/customisation/admin_templates.rst

```diff
@@ -91,6 +91,24 @@ To replace the welcome message on the dashboard, create a template file ``dashbo
 
     {% block branding_welcome %}Welcome to Frank's Site{% endblock %}
 
+Custom branding colors
+======================
+
+To customize the primary color used in the admin UI, inject a css file using the hook :ref:`insert_global_admin_css` and override the variables within the ``:root`` selector:
+
+.. code-block:: css
+
+    :root {
+        --color-primary-hue: 25;
+    }
+
+``color-primary`` is an `hsl color <https://en.wikipedia.org/wiki/HSL_and_HSV>`_ composed of 3 css variables - ``--color-primary-hue`` (a unitless number), ``--color-primary-saturation`` (a percentage), and ``--color-primary-lightness`` (also a percentage). Separating the color into 3 allows us to calculate variations on the color to use alongside the primary color. If needed, you can also control those variations manually by setting ``hue``, ``saturation``, and ``lightness`` variables for the following colors: ``color-primary-darker``, ``color-primary-dark``, ``color-primary-darkest``, ``color-input-focus``, and ``color-input-focus-border``.
+
+Warnings
+--------
+1. CSS variables are not supported in Internet Explorer, so the admin will appear with the default colors when viewed in that browser.
+2. The default Wagtail colors conform to the WCAG2.1 AA level color contrast requirements. When customizing the admin colors you should test the contrast using tools like `Axe <https://www.deque.com/axe/browser-extensions/>`_.
```

Comment:
Please use the warning directive here.

```
Warnings
---------
```
=>
```
.. warning::
    This is warning text. 
```
Just use two times the warning directive. No numbering.